pkg/tensorflow-lite/Makefile.tensorflow-lite-c

```diff
@@ -0,0 +1,9 @@
+MODULE = tensorflow-lite-c
+
+CFLAGS += -Wno-strict-prototypes
+CFLAGS += -Wno-old-style-definition
+
+NO_AUTO_SRC := 1
```

Comment:
```suggestion
SUBMODULES = 1
```

Reply:
`SUBMODULES` does more stuff, but I don't think it needs the `:`

Reply:
All fixed